utils/utils.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+// Parse environment variable and return boolean representation of string, or default value if environment variable is unset
+func GetBoolAsStringEnvVar(env string, defaultVal bool) (bool, error) {
```

Comment:
nit: as mentioned above, after thinking about this more, I think the caller code will be much cleaner if you pass the logger as an argument and log here when `ParseBool` returns an error